bokeh/charts/_charts.py

```diff
@@ -144,17 +147,28 @@ def start_plot(self, xgrid, ygrid):
         if ygrid:
             self.make_grid(1, yaxis.ticker)
 
-        # Add tools
+        # Add tools if supposed to
         if self.tools:
+            # only create pan and zoom tools if chart is not categorical...
+            if self.categorical:
+                tools_to_remove = ['pan', 'zoom']
+            else:
+                tools_to_remove = []
```

Comment:
One last note, I'd maybe suggest a different name. There is already the `add_tools` method on plot, and the main purpose of this function is to process strings into tool objects. Maybe this:
```
    tools = _process_tools_arg(plot, self.tools)
    p.add_tools(tools)
```